src/index.js

```diff
@@ -11,6 +14,10 @@ const NAME = __NAME__;
 
 export { Youtube as Engine };
 export { VERSION, NAME };
+export { YouTubePlugin as Plugin };
+
+const pluginName: string = 'youtube';
```

Comment:
why is registerplugin needed ?

Reply:
because we need to register youtube as a plugin in order to add chapters. the engine should not invoke player api to get timeline service.